### ISSUE TITLE
missing alt text

### ISSUE THREAD

**Issue by discospiff** [Neranjan98/CityOfCincy#16]
alt text should describe the image, for someone who can't see the image.  What colors are used?  What is in the image?  How big is the image?

https://github.com/Neranjan98/CityOfCincy/blob/fe9d66ac6f5751f59b224f92e09d61403dd202f1/CityOfCincy/Pages/Departments.cshtml#L37
